src/main/io/beeper.c

```diff
@@ -69,15 +69,15 @@
 #endif
 
 #ifdef USE_BEEPER
-#ifndef BEEPER_PWM_HZ
-#define BEEPER_PWM_HZ   0
-#endif
+# ifndef BEEPER_PWM_HZ
```

Comment:
Personal do not like indentation for macro's.

Reply:
Ok, I'll remove it. But it is confusing with deeper nesting

Reply:
Agree but could not find formal code guidelines on macro indentation as the code base uses them flat all over.

Reply:
Actually there are 61 lines with indented macros. But it seems that I had written almost all of them and nobody was adventurous enough to change them back ;-)